client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -14,9 +14,21 @@ import { launchSiteOrRedirectToLaunchSignupFlow } from 'state/sites/launch/actio
 import { localizeUrl } from 'lib/i18n-utils';
 import { verifyEmail } from 'state/current-user/email-verification/actions';
 
+// A list of known tasks for the calypso client/state/data-layer/wpcom/checklist/index.js
+// You need to alter this object in case of adding / removing tasks
+export const SITE_CHECKLIST_KNOWN_TASKS = {
```

Comment:
This appears to have broken the Jetpack Checklist on My Plan, since I think those tasks weren't registered.

<img width="1196" alt="Screenshot 2020-07-11 at 10 04 48" src="https://user-images.githubusercontent.com/43215253/87220685-f9e30100-c35d-11ea-9912-fdd0b1d35e42.png">

`isPlaceholder` is always returning `true` as a result of this, since there are no tasks being registered. 

https://github.com/Automattic/wp-calypso/blob/6d7b0b5526e4b6b51309b4080d7e35a5adaeb59f/client/my-sites/plans/current-plan/jetpack-checklist/index.tsx#L150

Reply:
Great finding @Aurorum and thanks for the heads up! I opened a follow up issue to be discussed and resolved over there.

Reply:
Same here! This PR introduced that issue with the placeholder into production. Do we have a PR that will fix it yet?

Reply:
Transferred here https://github.com/Automattic/wp-calypso/issues/44099